.github/workflows/ci.yml

```diff
@@ -74,6 +74,7 @@ jobs:
           TFC_RUN_TASK_URL: "https://httpstat.us/200"
           GITHUB_POLICY_SET_IDENTIFIER: "hashicorp/test-policy-set"
           GITHUB_REGISTRY_MODULE_IDENTIFIER: "hashicorp/terraform-random-module"
+          OAUTH_CLIENT_GITHUB_TOKEN: "${{ secrets.OAUTH_CLIENT_GITHUB_TOKEN }}"
```

Comment:
Would it make sense to add a test scenario demonstrating when this fails, and another demonstrating how this fix is the solution to those failing test(s)?